library/bdBank/Deferred/User.php

```diff
@@ -69,4 +75,26 @@ protected function _fetchSum($formula, $where)
 
         return bdBank_Helper_Number::add($liveValue, $archiveValue);
     }
+
+    protected function _fetchCredit($formula, $where)
```

Comment:
Let's modify `_fetchSum` and remove `_fetchCredit`. I think we can do something like:

```sql
SELECT transaction_type, SUM(amount), SUM(tax_amount)
FROM xf_bdbank_transaction
WHERE to_user_id = 123 AND reversed= 0
GROUP BY transaction_type
```

Then do some more calculation in PHP according to transaction type.

Reply:
done via d1e7cd33149f80d29e4eae373a67e8a156360a0d